src/app/content/highlights/components/CardWrapper.spec.tsx

```diff
@@ -6,11 +7,37 @@ import createMockHighlight from '../../../../test/mocks/highlight';
 import * as domUtils from '../../../domUtils';
 import { Store } from '../../../types';
 import { assertDocument, remsToPx } from '../../../utils';
+import { assertWindow } from '../../../utils/browser-assertions';
 import { clearFocusedHighlight, focusHighlight } from '../actions';
-import { cardMarginBottom } from '../constants';
+import { cardMarginBottom, highlightKeyCombination } from '../constants';
 import Card from './Card';
 import CardWrapper from './CardWrapper';
 
+const dispatchKeyDownEvent = (
+  window: Window,
+  element: Document | HTMLElement,
+  key: string,
+  target?: HTMLElement
+) => {
+  const keyboardEvent = window.document.createEvent('KeyboardEvent');
+  keyboardEvent.initKeyboardEvent('keydown', true, true, window, key, 0, '', false, '');
```

Comment:
mdn says this function is deprecated, is there another approach we should use?

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/initKeyboardEvent

edit: nvm this is a spec, i don't care